build.gradle.kts

```
@@ -1,8 +1,22 @@
 plugins {
     java
+    application
     kotlin("jvm") version "1.9.23"
 }
 
+application {
+    // NOTE: In case you choose a different main class, make sure to update this path
+    mainClass = "george.backend.hiring.KotlinApplicationKt"
```

ℹ️ The default points to Kotlin main, if candidates want to package Java code, they can simply change this to:
```suggestion
    mainClass = "george.backend.hiring.JavaApplication"
```

---

Maybe add this also as a comment, that you can comment in the other one? :)

---

Could we maybe auto discover it if there is only one?